scripts/get_counts.py

```diff
@@ -431,6 +437,25 @@ def ParseMemoryString(s):
     else:
         return (int(s), '')
 
+def DivideMemory(total, n):
+    (value, unit) = ParseMemoryString(total)
+    sub_memory = value / n
+    if sub_memory != float(value) / n:
+        if unit in ['K', '']:
```

Comment:
Are you considering the possibility that the memory string had lower-case letters?